packages/components/tests/vue/components/tooltip.test.js

```diff
@@ -50,16 +49,16 @@ describe('tooltip.test.js', () => {
   })
 
   it('1 Tooltip - renders a vue instance (min requirements)', () => {
-    let cmp = mount(Tooltip, {
+    const cmp = mount(Tooltip, {
       propsData: {
         tooltipText: ''
       }
     })
     expect(cmp.isVueInstance()).toBeTruthy()
   })
 
-  it('2 Tooltip - renders a vue instance (min requirements)', () => {
-    let storeLocal = new Vuex.Store({
+  it('2 Tooltip - renders a vue instance (min requirements)', async () => {
```

Comment:
are the changes to this the fix for #534 ?

Reply:
I was not aware of #534 when I was making this change but I think it should fix it. I just discovered that `tooltip.test.js` is hanging even though all the tests succeeds and I decided to fix that.